packages/guides/integrating-the-safe-core-sdk.md

```diff
@@ -0,0 +1,353 @@
+
+* **Using ethers.js**
+
+  We can use the class `EthersAdapter` from `@gnosis.pm/safe-core-sdk` as a wrapper of `ethers.js`.
+
+  ```js
+  import { EthersAdapter } from '@gnosis.pm/safe-core-sdk'
+  import { ethers } from 'ethers'
+
+  const web3Provider = // ...
+  const provider = new ethers.providers.Web3Provider(web3Provider)
+  const safeOwner = provider.getSigner(0)
+
+  const ethAdapter = new EthersAdapter({
+    ethers,
+    signer: safeOwner
+  })
+  ```
+
+* **Using web3.js**
+
+  We can use the class `Web3Adapter` from `@gnosis.pm/safe-core-sdk` as a wrapper of `web3.js`.
+
+  ```js
+  import { Web3Adapter } from '@gnosis.pm/safe-core-sdk'
+  import Web3 from 'web3'
+
+  const ethAdapter = new Web3Adapter({
+    web3,
+    signerAddress: safeOwnerAddress
+  })
+  ```
+
+Once we have an instance of `EthersAdapter` or `Web3Adapter` we are ready to instantiate the `SafeFactory` and `Safe` classes from `@gnosis.pm/safe-core-sdk`.
+
+```js
+import Safe, { SafeFactory } from '@gnosis.pm/safe-core-sdk'
+
+const safeFactory = await SafeFactory.create({ ethAdapter })
+
+const safeSdk = await Safe.create({ ethAdapter, safeAddress })
+```
+
+There are two versions of the Safe contracts: [GnosisSafe.sol](https://github.com/gnosis/safe-contracts/blob/v1.3.0/contracts/GnosisSafe.sol) that does not trigger events in order to afford some gas and [GnosisSafeL2.sol](https://github.com/gnosis/safe-contracts/blob/v1.3.0/contracts/GnosisSafeL2.sol) that does it and it is more appropriate for L2 networks.
+
+By default `GnosisSafe.sol` will be only used on Ethereum Mainnet. For the rest of the networks where the Safe contracts are already deployed, the `GnosisSafeL2.sol` contract will be used unless you add the property `isL1SafeMasterCopy` to force the use of the `GnosisSafe.sol` contract.
+
+```js
+const safeFactory = await SafeFactory.create({ ethAdapter, isL1SafeMasterCopy: true })
+
+const safeSdk = await Safe.create({ ethAdapter, safeAddress, isL1SafeMasterCopy: true })
+```
+
+If the Safe contracts are not deployed in your current network, the property `contractNetworks` will be required to point to the addresses of the Safe contracts previously deployed by you.
+
+```js
+import { ContractNetworksConfig } from '@gnosis.pm/safe-core-sdk'
+
+const id = await ethAdapter.getChainId()
+const contractNetworks: ContractNetworksConfig = {
+  [id]: {
+    multiSendAddress: '<MULTI_SEND_ADDRESS>',
+    safeMasterCopyAddress: '<MASTER_COPY_ADDRESS>',
+    safeProxyFactoryAddress: '<PROXY_FACTORY_ADDRESS>'
+  }
+}
+
+const safeFactory = await SafeFactory.create({ ethAdapter, contractNetworks })
+
+const safeSdk = await Safe.create({ ethAdapter, safeAddress, contractNetworks })
+```
+
+The `SafeFactory` constructor also accepts the property `safeVersion` to specify the Safe contract version that will deploy. This string can take the values `1.1.1`, `1.2.0` or `1.3.0` and if not specified the last version will be used by default.
```

Comment:
"...`1.3.0`. If not specified, the most recent contract version will be used by default."